### PR TITLE
#310 Support stereo CIP calculation in Ket format

### PR DESCRIPTION
**Pull request by even1024**

### CODE REVIEW

File: core/indigo-core/molecule/src/molfile_saver.cpp [hunk 1]
```diff
@@ -149,6 +149,7 @@ void MolfileSaver::_saveMolecule(BaseMolecule& mol, bool query)
 
     MoleculeCIPCalculator mcc;
     mcc.updateCIPStereoDescriptors(mol, add_stereo_desc);
+    mcc.addCIPSgroups(mol);
```
Comment (AlexanderSavelyev) [inline core/indigo-core/molecule/src/molfile_saver.cpp:152]: s-group should be added only if option is on

**Review by AlexanderSavelyev — CHANGES_REQUESTED** [review]

File: core/indigo-core/molecule/src/molfile_saver.cpp (hunk 1 above)
Comment (even1024) [reply core/indigo-core/molecule/src/molfile_saver.cpp:152]: updateCIPStereoDescriptors adds corresponding properties to bonds and atoms if add_stereo_desc = true.
addCIPSgroups has not effect if no properties has been added.

File: core/indigo-core/molecule/src/molfile_saver.cpp [hunk 2]
```diff
@@ -149,6 +149,7 @@ void MolfileSaver::_saveMolecule(BaseMolecule& mol, bool query)
 
     MoleculeCIPCalculator mcc;
     mcc.updateCIPStereoDescriptors(mol, add_stereo_desc);
```
Comment (AlexanderSavelyev) [inline core/indigo-core/molecule/src/molfile_saver.cpp:151]: change the function to be more obvious
if (add_stereo_desc) {
MoleculeCIPCalculator mcc;
mcc.addCIP....()
mcc.addCIPSgroups(mol);
}

**Review by AlexanderSavelyev — CHANGES_REQUESTED** [review]

File: core/indigo-core/molecule/src/molfile_saver.cpp (hunk 2 above)
Comment (even1024) [reply core/indigo-core/molecule/src/molfile_saver.cpp]: fixed

**Review by AlexanderSavelyev — APPROVED** [review]

**Pull request merged**